src/Core/Protocol.h

```diff
@@ -52,6 +52,10 @@ namespace DB
 /// Using this block the client can initialize the output formatter and display the prefix of resulting table
 /// beforehand.
 
+/// Marker of the inter-server secret (passed in the user name)
+/// (anyway user cannot be started with a whitespace)
+const char INTERSERVER_SECRET_MARKER[] = " INTERSERVER SECRET IS ENABLED ";
```

Comment:
I think we can shorten the name. Just " INTERSERVER SECRET " seems to be enough.